### PR TITLE
Initial recognition example with openSUSE button

### PR DESCRIPTION
**Pull request by lkocman**
* This is just an example, I'd really appreciate more personalized
graphics.

* Done per ovationcore's example which will be used for SUSE Employees

* The list of recipients https://bit.ly/2Q6nTqe

### CODE REVIEW

**Comment by hellcp** [comment]
We have that unused leap tshirt design that we can reuse for graphics too https://progress.opensuse.org/issues/63823

File: recognition/recognition_example_individual.htm [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+                    </tr>
+                    <tr width="530">
+                        <td align="left" style="font-family: Arial, sans-serif; font-size:16px; color:#000; padding:0px;" valign="top" width="530">
+                        <table border="0" cellpadding="10" cellspacing="0" style="margin-bottom:10px;" width="530">
+                            <tbody>
+                                <tr>
+                                    <td align="left" style="font-family: Arial, sans-serif; font-size:16px; color:#000;" valign="top">
+                                    <p style="font-family: Arial, sans-serif; color:#000; font-size:16px; line-height:24px;"><p>Dear __USER__</p>
+
+<p>You have been recognized by openSUSE Leap users as an excaptional contributor to the openSUSE Leap 15.2 release in our <a href="https://en.opensuse.org/Portal:15.2/Retrospective">release retrospective</a>.<strong>
```
Comment (victorhck) [inline recognition/recognition_example_individual.htm:22]: excaptional

Maybe a typo?

**Review by victorhck — COMMENTED** [review]
Seems to be fine, but the typo! :)

**Comment by tigerfoot** [comment]
Almost an **exceptional** work
a ) Would you mind to fix the following error (mentionned by victorkhck) excaptional has to be written exceptional
b) Why do you broke the indentation of the paragraph ? would be better to have it correctly done.

File: recognition/recognition_example_individual.htm (hunk 1 above)
Comment (lkocman) [reply recognition/recognition_example_individual.htm:22]: Will fix, thank you!

**Comment by lkocman** [comment]
Perhaps the quote "You took Leap to 15.2" just like on t-shirt sounds more cool than "Thanks for making Leap 15.2 happen". Sounds less of a sentence generated by buzzword generator :-)

**Comment by tigerfoot** [comment]
Your last comment bring up a nice moto 

I've a last thing that bother me "font-family: Arial" ... 
Couldn't we replace that by some font normal people have like noto or any other included font on our *.opensuse.org website ?

**Comment by lkocman** [comment]
> u

Thank you tigerfoot, I agree about that dark color for the border will work better with the current banner. Font size could be also increased.

**Comment by lkocman** [comment]
commit 7c40217 looks lovely, larger fonts (18px) matching border and a letter spacing to move text further from border. This already looks pro. I also added a recognition for people who were not mentioned in retro but still deserve it.
@hellcp @victorhck could you please merge it unless you have some comments or remarks?

final

**Pull request merged**